src/Compilers/CSharp/Test/Semantic/Semantics/NameOfTests.cs

```diff
@@ -9,6 +9,7 @@
 using Xunit;
 using System.Threading;
 using System.Linq;
+using static Microsoft.CodeAnalysis.CSharp.UnitTests.SymbolUtilities;
```

Comment:
Consider reverting this file to its previous state because it looks like this is the only change in it.